mods/ra/rules/infantry.yaml

```diff
@@ -195,6 +195,9 @@ E3:
 		Weapon: Dragon
 		LocalOffset: 0,0,555
 	Armament@GARRISONED:
+		Name: garrisoned
+		Weapon: RedEye
```

Comment:
This is a balance change. The anti-ground bunker suddenly becomes AA.

Reply:
I've also seen it tested in quite a lot of older balance maps, its fairly weak without additional buffs

Reply:
I would like to comment that this seems like a good idea. It makes the game more consistent logically and the AA unbalancing would be a minor effect if at all.